### DISCUSSION TITLE
Anbindung Victron

### DISCUSSION THREAD

**Discussion by peff74** [hoylabs/OpenDTU-OnBattery#211] [Q&A] [answered]
Mahlzeit,

ich versuche gerade zu verstehen ob es einen Sinn machen würden meinen Victron 100/20 mit einzubinden.
Würden die Daten die er liefert in irgendeiner Form mit in die Steuerung der Dynamic Power Limiter Funktion eingreifen?
Wir darüber ggf. die Solar Pass-though Funktion geregelt?

Danke für Infos.

Gruss
Peff

P.S. 
die Dynamic Power limiter Funktion scheint jetzt in der neuen Version besser zu laufen.

**Comment by helgeerbe** [comment]
@peff74 macht Sinn, und ja, für Solar Passthrough brauchst due den Victron da man ja wissen muss, wie viel Leistung die Panels gerade liefern können. Aber grundsätzlich ist es auch schon gut zu wissen, in welchem Zustand sich dein Ladegerät befindet.

**Reply by peff74** [reply]
Also hat es keinen Einfluss den Haken PassThough zu setzen, wenn der Victron nicht angebunden ist?

Auf den Laderegler greife ich z.Z. noch über ein Handy mit Teamviewer zu. Klar will ich ja wissen was abgeht.
Die Anlage is ja mal gerade 5 Tage aktiv ;-)

**Reply by helgeerbe** [reply, marked as answer]
Nein, dann wird an den PowerLimiter immer zurückgegeben, dass kein Sonnenlicht da ist. Es gibt ja keinen Meßwert.

**Comment by ar12321** [comment]
Hallo,
ist der Victron Lastausgang in Abhängigkeit vom Ladezustand / über FHEM schaltbar?
